GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,168 @@
+  * Have a hypervisor installed (kvm  is recommended for linux)
+
+Alternatively, you can install it using podman or docker driver which will launch the minikube cluster inside the containers.
+Finally for installing the Minikube cluster please follow [this tutorial](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+
+## Build Artifacts
+Before proceeding read the README files on the following repositories to know the dependencies you might need to install to build and deploy the artifacts.
+ * [kogito-runtimes](https://github.com/kiegroup/kogito-runtimes)
+ * [kogito-apps](https://github.com/kiegroup/kogito-apps)
+ * [kogito-examples](https://github.com/kiegroup/kogito-examples)
+ 
+For generating the jar artifacts you can use `mvn clean install` to have them locally on your system.
+
+So for every repository, [runtimes](https://github.com/kiegroup/kogito-runtimes) / [apps](https://github.com/kiegroup/kogito-apps) / [examples](https://github.com/kiegroup/kogito-examples). From the root of the repository you can just run:
+
+```shell-script
+mvn clean install -DskipTests #(For skipping tests)
+```
+
+This will deploy your artifacts in `~/.m2/repository/org/kie/kogito`. You can just build the artifacts you modified or added new feature in and update them in the Images
+
+## Build the Kogito Application Images
+We’re going to use our pre-built images which are available in the `quay.io/kiegroup/` namespace.
+
+ ### Images list
+  * kogito-data-index
+  * kogito-jobs-service
+  * kogito-management-console
+
+Now let’s take an example of updating the `kogito-jobs-service` artifact in the image.
+Create a directory e.g: `~/images/`
+Inside the directory copy the artifacts you need to update,
+
+```shell-script
+$ cp ~/.m2/repository/org/kie/kogito/jobs-service/8.0.0-SNAPSHOT/jobs-service-8.0.0-SNAPSHOT-runner.jar .
+```
+
+Inside the directory you can have a `Dockerfile` which can be used to update the artifact.
+
+` $ vim jobs-service-Dockerfile` 
+
+```
+FROM quay.io/kiegroup/kogito-jobs-service
+COPY jobs-service-8.0.0-SNAPSHOT-runner.jar ${KOGITO_HOME}/bin/kogito-jobs-service-runner.jar
+```
+**Note**: Normally all the latest artifacts would be in `8.0.0-SNAPSHOT`
+**Note**: You can check the location of the artifact in the image from the [kogito-images](https://github.com/kiegroup/kogito-images) repository.
+
+Now just build your image. Inside the above created directory run:
+
+`$ podman build -t <prefered-tag-name>  -f jobs-service-Dockerfile  .`
+
+The above command will build your image with `<prefered-tag-name>` and can be seen in the output of `$ podman images`
+
+**Note**: We added the `-f` flag to specify the `Dockerfile` in case you want to build multiple images, you can then have different files and specify them with the `-f` flag.
+This image needs to be on a public container registry from which it can be pulled later.
+```shell-script
+$ podman tag <prefered-tag-name> quay.io/<your-namespace>/<image-name>
```

Comment:
why not calling it directly `quay.io/<your-namespace>/<image-name>` in the previous command ?